src/functionalities.js

```diff
@@ -0,0 +1,120 @@
+import updateStatus from './status.js';
+
+export default class Tasks {
+  constructor() {
+    this.tasksList = [];
+  }
+
+  removeTask(index) {
+    this.tasksList.splice(index, 1);
+    for (let i = index; i < this.tasksList.length; i += 1) {
+      this.tasksList[i].index = i;
+    }
+    localStorage.setItem('tasks', JSON.stringify(this.tasksList));
+    this.updateDisplay();
+  }
+
+  createTaskElement(taskObj) {
+    const item = document.createElement('li');
+    item.classList.add('list-item');
+    item.setAttribute('data-index', taskObj.index);
+    if (taskObj.completed === false) {
+      item.innerHTML = `<button><i class="fa-regular fa-square checkbox"></i></button>
+        <input type="text" value="${taskObj.description}" class="task-input">
+        <i class="fa-solid fa-ellipsis-vertical trash-can"></i>`;
+
+      const taskInput = item.querySelector('.task-input');
+      const trashCan = item.querySelector('.trash-can');
+
+      taskInput.addEventListener('focus', () => {
+        item.style.backgroundColor = 'rgb(205, 187, 205)';
+        trashCan.classList.remove('fa-solid');
+        trashCan.classList.remove('fa-ellipsis-vertical');
+        trashCan.classList.add('fa-regular');
+        trashCan.classList.add('fa-trash-can');
+        trashCan.style.cursor = 'pointer';
+        trashCan.addEventListener('click', () => {
+          const index = parseInt(trashCan.parentElement.getAttribute('data-index'), 10);
+          this.removeTask(index);
+          trashCan.parentElement.remove();
+        });
+      });
+
+      taskInput.addEventListener('focusout', () => {
+        item.style.backgroundColor = 'white';
+        trashCan.classList.add('fa-solid');
+        trashCan.classList.add('fa-ellipsis-vertical');
+        trashCan.classList.remove('fa-regular');
+        trashCan.classList.remove('fa-trash-can');
+        trashCan.style.cursor = 'move';
+      });
+
+      taskInput.addEventListener('keypress', (e) => {
+        if (!e) e = window.event;
+        const keyCode = e.code || e.key;
+        if (keyCode === 'Enter') {
+          e.preventDefault();
+          const index = taskInput.parentElement.getAttribute('data-index');
+          this.tasksList[index].description = taskInput.value;
+          localStorage.setItem('tasks', JSON.stringify(this.tasksList));
+        }
+      });
+
+      taskInput.addEventListener('keyup', (e) => {
+        if (e.keyCode === 13) {
+          e.preventDefault();
+          e.target.blur();
+        }
+      });
+    } else {
+      item.innerHTML = `<button><i class="fa-regular fa-square-check checkbox checkbox-ticked"></i></button>
+        <span class="task-span">${taskObj.description}</span>
+        <i class="fa-solid fa-ellipsis-vertical trash-can"></i>`;
```

Comment:
Considering Lines 22-24, This is unnecessary repetition. I notice a checkbox attribute as the main difference. I suggest you  implement a simple function to handle this difference while avoiding duplication. This will ensure you Don't Repeat yourself (DRY) in your codebase.